randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -417,8 +435,11 @@ Extra - asset_id: 11
       All of the following:
           Morph Ball
           Any of the following:
-              Morph Ball Bombs or Power Bombs ≥ 4
-              Power Bombs and Screw Attack
+              Power Bombs ≥ 4 or Can Lay Bombs
```

Comment:
It'd be nice to have "Can Lay Power Bombs AND PB >= 4" for all of these

Reply:
Updated the 4 connectins in Gauntlet Entrance and Gauntlet Etank Room. The all now have morph, pbs >= x, and canlaypowerbombs